highway_env/__init__.py

```diff
@@ -97,6 +97,10 @@ def _register_highway_envs():
         id="racetrack-v0",
         entry_point="highway_env.envs.racetrack_env:RacetrackEnv",
     )
+    register(
+        id="racetrack-v0",
```

Comment:
shouldn't this use a different id, like racetrack-large-v0?

Reply:
True!